### ISSUE TITLE
No hands or digits on clock?

### ISSUE THREAD

**Issue by fribse** [Villhellm/lovelace-clock-card#20]
HA: 2022.9.7
Lovelace Clock Card: 0.4.1
Firefox 105.0.1 (64-bit) ( Works on Chrome 105.0.5195.12)

Really weird, I can't figure out what is happening.
When I'm in the editor the clock shows nicely:
![billede](https://user-images.githubusercontent.com/13660621/192707969-36129fb7-2090-4826-9870-e94734f4ddf0.png)
But in the Lovelace interface it doesn't show the hands or digits?
![billede](https://user-images.githubusercontent.com/13660621/192708033-9d17d05e-09d6-4402-9d99-2289c22e2867.png)

**Comment by fribse** [comment]
I did a complete cleanup of Firefox, all cookies and offline cached files deleted, and now they digits and hands show up again.

**Comment by fribse** [comment]
Nope, on the recent versions of HA, the hour hand isn't showing on Firefox. It seems to work in Chrome.
If I clear the cache in Firefox, it works for a short while.

**Comment by BebeMischa** [comment]
I'm facing the same problem ;-)

**Comment by BebeMischa** [comment]
Looks like @Villhellm left the building... :-(

**Comment by breti** [comment]
I read that Villhellm has unfortunately passed away :-(

I have implemented a workaround in my fork. Unfortunately I'm not a javascript developer and don't know the cause, but at least this way the clock works again.

https://github.com/breti/lovelace-intl-clock-card

Edit: https://www.home-assistant.io/blog/2021/01/06/release-20211/#in-memoriam-of-villhellm
